pkgs/tools/misc/dvtm/default.nix

```diff
@@ -1,20 +1,28 @@
-{ stdenv, fetchurl, ncurses }:
+{ stdenv, fetchurl, ncurses, customConfig ? null }:
 
 stdenv.mkDerivation rec {
+
   name = "dvtm-0.15";
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Dynamic virtual terminal manager";
     homepage = http://www.brain-dump.org/projects/dvtm;
-    license = stdenv.lib.licenses.mit;
-    platfroms = stdenv.lib.platforms.linux;
+    license = licenses.mit;
+    platfroms = platforms.linux;
+    maintainers = [ maintainers.vrthra ];
   };
 
   src = fetchurl {
     url = "${meta.homepage}/${name}.tar.gz";
     sha256 = "0475w514b7i3gxk6khy8pfj2gx9l7lv2pwacmq92zn1abv01a84g";
   };
 
+  patchPhase = stdenv.lib.optionalString (customConfig != null) ''
```

Comment:
Does `dtvm` encourage customization via patching? If so, we might at some point want to add a `patches` parameter and have them applied during the standard patch phase. To allow for that, I think it'd be more appropriate to write out the config.h in `postPatch` or somesuch rather than overwriting the patch phase entirely.